series_functions/yaxis.js

```diff
@@ -27,21 +30,35 @@ module.exports = new Chainable('yaxis', {
       types: ['string', 'null'],
       help: 'left or right'
     },
+    {
+      name: 'units',
+      types: ['string', 'null'],
+      help: 'The function to use for formatting y-axis labels. One of: ' + _.values(tickFormatters).join(', ')
+    },
   ],
   help: 'Configures a variety of y-axis options, the most important likely being the ability to add an Nth (eg 2nd) y-axis',
   fn: function yaxisFn(args) {
-    return alter(args, function (eachSeries, yaxis, min, max, position) {
+    return alter(args, function (eachSeries, yaxis, min, max, position, units) {
       yaxis = yaxis || 1;
-
       eachSeries.yaxis = yaxis;
       eachSeries._global = eachSeries._global || {};
 
       var yaxes = eachSeries._global.yaxes = eachSeries._global.yaxes || [];
+      _.times(yaxis-1,function(idx){
```

Comment:
Clean up the linting in this file, there's a `.eslintrc` in the root of the project